Source/VMware.vSphereDSC/DSCResources/VDPortGroup.ps1

```diff
@@ -129,11 +160,13 @@ class VDPortGroup : BaseDSC {
         }
         finally {
             $this.DisconnectVIServer()
+            Write-VerboseLog -Message $this.TestMethodEndMessage -Arguments @($this.DscResourceName)
```

Comment:
Here as well **Write-VerboseLog** will need to be replaced by **WriteLogUtil** after the other PR gets approved.